calico-cloud/operations/cluster-management.mdx

```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
 
 To connect new clusters to {{prodname}}, select Managed Clusters in the left nav, and click **Connect Cluster**.
 
-## Recover a deleted managed cluster
+## Delete a managed cluster
 
-If you inadvertenty delete a managed cluster, rerun the `curl` command and the cluster will reconnect to {{prodname}}.
+### Prerequisites
+
+- Only users with either the `Owner` or `Admin` [roles](../users/user-management.mdx) are permitted to delete managed clusters.
```

Comment:
* You are signed in to Manager UI and you have [Owner or Admin privileges](../users/user-management.mdx).

Reply:
Item updated https://github.com/tigera/docs/pull/961/commits/f11f949cc5a80c156000fba8ec494143384c5a10